mod-source-record-manager-server/src/main/java/org/folio/verticle/DataImportJournalBatchConsumerVerticle.java

```diff
@@ -421,11 +421,12 @@ private Completable commitOffset(Map<TopicPartition, OffsetAndMetadata> offsets)
       )
       .onErrorResumeNext(error -> {
         if (error instanceof RebalanceInProgressException) {
-          LOGGER.warn("Rebalance in progress. Retrying offset commit...");
-          return kafkaConsumer.rxPoll(Duration.ofMillis(100))
-            .flatMapCompletable(records -> commitOffset(offsets));
+          LOGGER.warn("Rebalance in progress. Waiting for the re-balance to complete...");
+          return Completable.timer(1, TimeUnit.MILLISECONDS);
```

Comment:
This is one millisecond instead of one second?

Reply:
fixed :)